clang/lib/Driver/Driver.cpp

```diff
@@ -169,6 +169,17 @@ getHIPOffloadTargetTriple(const Driver &D, const ArgList &Args) {
   return std::nullopt;
 }
 
+
+static std::optional<llvm::Triple>
+getINTELOffloadTargetTriple(const Driver &D, const ArgList &Args,
+                             const llvm::Triple &HostTriple) {
+  if (!Args.hasArg(options::OPT_offload_EQ)) {
+    return llvm::Triple(HostTriple.isArch64Bit() ? "spirv64-intel-sycl"
+                                                 : "spirv32-intel-sycl");
+  }
+  return std::nullopt;
```

Comment:
Should we emit something if user specifies -offload= for SYCL offloading? Or atleast add an assert?